pkg/kyverno/test/test_command.go

```diff
@@ -577,32 +599,61 @@ func applyPoliciesFromPath(fs billy.Filesystem, policyBytes []byte, valuesFile s
 	engineResponses := make([]*response.EngineResponse, 0)
 	var dClient *client.Client
 	values := &Test{}
+	versionedvalues := &versionedTest{}
+	versioned := true
 	var variablesString string
 	var pvInfos []policyreport.Info
 	var resultCounts common.ResultCounts
+	var resultsMap map[string]report.PolicyReportResult
+	var testResults []TestResults
+	var policyFullPath []string
+	var resourceFullPath []string
 	store.SetMock(true)
 
-	if err := json.Unmarshal(policyBytes, values); err != nil {
+	if err := json.Unmarshal(policyBytes, versionedvalues); err != nil {
```

Comment:
Has this been tested with the old format? Seems pretty like `Unmarshal` could error out on the old format.

Reply:
Hey @MarcelMue I have tested this on the old format as well as the new format. It works as expected.